doc/manual/manual.tex

```diff
@@ -6795,8 +6795,8 @@ \subsubsection{3D convection with an Earth-like initial condition}
 \end{figure}
 
 
-\marginpar{Use cookbooks/periodic-box.prm}
-\marginpar{Finish the GPlates section}
+%\marginpar{Use cookbooks/periodic-box.prm}
+%\marginpar{Finish the GPlates section}
```

Comment:
Yes, there are a couple of marginpars that have survived essentially the entire history of ASPECT. We did not have an issue tracker in the beginning that would have allowed us to track notes such as these outside the manual. They should not be in the manual any more these days.

Feel free to remove these two -- I think they really are resolved -- and remove others in a separate patch if you open an issue for each marginpar that is still relevant. Just don't comment them out -- that doesn't help anyone :-)